cmd/dockerd/daemon_unix.go

```diff
@@ -19,7 +19,7 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-const defaultDaemonConfigFile = "/etc/docker/daemon.json"
+const defaultDaemonConfigFile = "/etc/docker/daemon.toml"
```

Comment:
adding support is one thing, but we can't really change the default

Reply:
Yes I agree with what you're saying. I changed it to show @AkihiroSuda that it's not a problem to set `toml` to be the defaults. https://github.com/moby/moby/pull/34838#issuecomment-329400918

Reply:
I meant we should check the existence of both toml and json.

probably:
```go
var defaultDaemonConfigSearchPaths = ["/etc/moby-engine/daemon.toml", "/etc/moby-engine/daemon.json", "/etc/docker/daemon.toml", "/etc/docker/daemon.json"]
```
(Ping @cpuguy83 @tiborvass for file names)

Reply:
Oh, no problem. It won't be hard to implement.

Reply:
I looked at it closer and to turn `defaultDaemonConfigFile` into `defaultDaemonConfigSearchPaths` it will add more code which is out of this patch scope.

I will have to change the `config-file` option to get a list of potential config file pathes and loop over that list till a file under the given path exists.

It's not a problem to implement, I am just asking, do you want me to add this to this patch?

Thanks.